### ISSUE TITLE
Export XDD file - Memory Types are not stored

### ISSUE THREAD

**Issue by martinwag** [robincornelius/libedssharp#139]
Hi,

when exporting a CANopen xml to XDD the memory types seem to be lost. When importing a XDD file, the memory types seem to be assinged at random (?).

**Comment by martinwag** [comment]
This seems to apply also for the OD entries marked as not "Enabled".
What are your plans regarding XDD? Should this replace the CANopenNode xml format or remain an additional export/import option?
As far as I can see, my CO_OD.c/h otherwise matches from both file formats.

**Comment by robincornelius** [comment]
xdd ideally should replace it as its the CanOpen offical standard, but i'll need to take a look to see if there are any easy ways to "extend" it with these extra attributes.

**Comment by robincornelius** [comment]
I've added memory type support to XDD, so now its pretty complete. I believe the main file format for the app should be XDD (for individual files anyway) I'm going to change it to default save to XDD, but Save As/Export .. all the other formats. I'll probably create an array of XDD containers to make the network format also work in XDD. But i'm not going to remove importing/exporting any of the old formats there is too much old stuff using them. I've got loads my self and i'm sure everyone else does to
